src/app/providers/sessionRefreshProvider.tsx

```diff
@@ -1,14 +1,22 @@
 "use client";
 
 import { signOut, useSession } from "next-auth/react";
-import { useEffect } from "react";
+import { SplitsClientConfig, SplitsProvider } from "@0xsplits/splits-sdk-react";
+import { useEffect, useState } from "react";
+import { useChainId, usePublicClient, useWalletClient } from "wagmi";
 
 type Props = {
   children?: React.ReactNode;
 };
 
 export const SessionRefreshProvider = ({ children }: Props) => {
+  const [splitsConfig, setSplitsConfig] = useState<SplitsClientConfig | null>(
+    null,
+  );
   const { data: session } = useSession();
+  const { data: walletClient, status } = useWalletClient();
+  const publicClient = usePublicClient();
+  const chainId = useChainId();
```

Comment:
We should def move the splits config out of here.
This was for testing;
We need to create a splitsProvider.tsx file and import it in the same way that we import the sessionRefreshProvider;
This will also help us to avoid that issue, e.g. only load the splits provider when wagmi provider is already loaded on the parent.

We can work on the fix for that on the next PR, but I suggest that we already move this into a new file on this one. Your call